instrumentation/mongo/mongo-3.1/javaagent/src/test/groovy/MongoClientTest.groovy

```diff
@@ -37,6 +37,7 @@ class MongoClientTest extends MongoBaseTest {
   def "test create collection"() {
     setup:
     MongoDatabase db = client.getDatabase(dbName)
+    client.listDatabaseNames()
```

Comment:
Is this line needed?

Reply:
Ah, no. I accidentally left that in there. Cleaning it up.